pages/_app.js

```diff
@@ -65,6 +67,8 @@ class MyApp extends App {
         <Mdx>
           <Container>
             <Head>
+              <script src="https://cdn.polyfill.io/v2/polyfill.min.js" />
+              <style dangerouslySetInnerHTML={{ __html: styles }} />
```

Comment:
Minor, but perhaps we can just embed `styles` inside of `injectGlobal`.

Reply:
There's a perf hit if we do that. It was my first choice, but it resulted in a flash of unstyled content that was very noticeable. This is just temporary until we convert legacy scss to css-in-js

Reply:
👍